src/ActionList.fs

```diff
@@ -0,0 +1,54 @@
+module Polaris.ActionList
+
+open Fable.React
+open Fable.Core
+open Fable.Core.JsInterop
+open Polaris.Shared
+
+type [<StringEnum>] [<RequireQualifiedAccess>] ItemDescriptorBadgeStatus =
+    |  [<CompiledName "new">] New
+
+type [<RequireQualifiedAccess>] ActionListItemDescriptorBadge(content, ?status) =
+    member __.Content : string = content
+    member __.Status : ItemDescriptorBadgeStatus option = status
+
+type [<RequireQualifiedAccess>] ActionListItemDescriptor(content, ?accessibilityLabel, ?active, ?badge, ?destructive, ?disabled, ?ellipsis, ?external, ?helpText, ?icon, ?id, ?image, ?role, ?target, ?url, ?onAction) =
```

Comment:
what does the `?` do?

Reply:
So here I create a class that becomes sort-of like the DU in the end.
The ? are optional params, so this way we can enforce required params and optional ones without having to do what I used to do before (passing everything as an option with None as a value)

This allow to create simple elements like `ActionListItemDescriptor(content = "My content")` and not have to provide the rest

Reply:
understood